src/main/kotlin/graphql/kickstart/tools/resolver/MissingResolverDataFetcherProvider.kt

```diff
@@ -0,0 +1,14 @@
+package graphql.kickstart.tools.resolver
+
+import graphql.kickstart.tools.SchemaParserOptions
+import graphql.language.FieldDefinition
+import graphql.schema.DataFetcher
+
+/**
+ * Provider for missing resolver data fetchers.
+ */
+fun interface MissingResolverDataFetcherProvider {
+   fun createDataFetcher(field: FieldDefinition,
```

Comment:
Reformat to `fun createDataFetcher(field: FieldDefinition, options: SchemaParserOptions): DataFetcher<*>` please.